apps/snitch_core/mix.exs

```diff
@@ -60,7 +60,7 @@ defmodule Snitch.Core.Mixfile do
       {:ecto_autoslug_field, "~> 0.5"},
 
       # state machine
-      {:beepbop, github: "oyeb/beepbop", branch: "develop"},
+      {:beepbop, github: "anantanant2015/atom_beepbop", branch: "develop"},
```

Comment:
@anantanant2015  Why `atom_beepbop`, just keep it `beepbop` would be enough imo.

Reply:
done